### DISCUSSION TITLE
Fixing the <code> tag and website's scrollbar design.

### DISCUSSION THREAD

**Discussion by plutocrate** [TheOdinProject/theodinproject#3769] [Ideas]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `brief description of request` format, e.g. `Add dark mode to website`

The following checkbox is OPTIONAL:

-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

While reading the texts written at **code tag**  the scrollbar design looks a bit off and eye-harsh.

![Screenshot 2023-04-08 192641](https://user-images.githubusercontent.com/96922561/230725992-4d9fda50-f14e-4ca8-a663-99c3ac563a3f.png)

Fixing the design of the scrollbar will surely enhance the user experience. I tried to put a new color scheme and design to the scrollbar using chrome's dev tools(I used the color scheme provided by TOP). Moreover, we can follow the same scrollbar theme for each page's side scrollbar.

![Screenshot 2023-04-08 192807](https://user-images.githubusercontent.com/96922561/230726097-ea2edfc0-1b31-4931-b5ca-3953052a28d3.png)

**2. Acceptance Criteria:**

- [x] A theme toggle is present on the dashboard
- [x] Clicking the theme toggle changes between light and dark
- [x] A user's theme choice persists after leaving the website

**3. Additional Information:**

I did add the screenshots along with the issue I came across.

**Comment by KevinMulhern** [comment]
Thanks for the suggestion @prathammpurohit, styling the code block scroll bars is a good idea. 
But we wouldn't want to go with anything like the pink in the screenshots, it's too distracting. A subtle grey color that blends into the light/dark theme would be best.  

We would also prefer not to style the browser scroll bars and leave that to whatever users have set as their browser theme.
